packages/commons-ui-core/src/Copyright/Copyright.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+
+const Copyright = React.forwardRef(function Copyright(props, ref) {
+  const {
+    copyright,
+    CopyrightProps,
+    icon,
+    IconProps,
+    variant,
+    year,
+    YearProps,
+    url,
+    sx,
+  } = props;
+
+  return (
+    <Box sx={sx} ref={ref}>
+      {copyright && (
+        <Typography variant={variant} {...CopyrightProps}>
+          {copyright}
+        </Typography>
+      )}
+      {icon && (
+        <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
1. Shouldn't you check for presence of `icon` **and** `url`?
2. Don't we already have `ImageButton` (or similiar-ish components) to handle displaying clickable images?